src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -289,10 +289,14 @@ export class LabeledMenuItemActionItem extends MenuEntryActionViewItem {
 				}
 
 				if (this.label) {
-					this.label.classList.add('codicon', this._defaultCSSClassToAdd, ...iconClass.split(' '));
+					const iconClasses = iconClass.split(' ');
```

Comment:
I've noticed several exceptions in other places during code merge testing that `iconClass.split(' ')` throws, I think due to things like lead ' ' character.  I added guards to avoid, maybe not needed here, but something to be aware of.